Microsoft.NET.Build.Containers/CreateNewImageToolTask.cs

```diff
@@ -0,0 +1,153 @@
+namespace Microsoft.NET.Build.Containers.Tasks;
+
+using System;
+using System.Linq;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+/// <summary>
+/// This task will shell out to the net7.0-targeted application for VS scenarios.
+/// </summary>
+public class CreateNewImage : ToolTask
+{
+    /// <summary>
+    /// The path to the folder containing `containerize.dll`.
+    /// </summary>
+    [Required]
+    public string ContainerizeDirectory { get; set; }
```

Comment:
Nit for later refactor because it just occurred to me: make this + the other task a partial class, and pull all these input /output properties into a single file, to reduce maintenance.

Reply:
oh of course. F# doesn't have partial classes so they never jump out to me as a design possibility.

Reply:
https://github.com/dotnet/sdk-container-builds/issues/171